addons/mail/static/src/components/emoji_list/emoji_list.scss

```diff
@@ -3,20 +3,11 @@
 // ------------------------------------------------------------------
 
 .o_EmojiList {
-    display: flex;
-    flex-flow: row wrap;
-    max-width: 200px;
+    // Could be replaced by "d-grid" class after migration to BS5
+    display: grid;
+    grid-template-columns: repeat(9, 1fr);
 }
 
 .o_EmojiList_emoji {
-    font-size: 1.1em;
-    margin: map-get($spacers, 1);
-}
-
-// ------------------------------------------------------------------
-// Style
-// ------------------------------------------------------------------
-
-.o_EmojiList_emoji {
-    cursor: pointer;
+    font-size: $font-size-lg;
```

Comment:
would it better to wait for `fs-[x]` classes?

Reply:
Indeed, it could be better 👍 .
Ok let's wait, I will convert the PR to draft in the meantime.